cineast-core/src/main/java/org/vitrivr/cineast/core/extraction/segmenter/general/Segmenter.java

```diff
@@ -39,4 +40,10 @@
    */
   @Override
   void close();
+
+  /**
+   * Specifies for which media type this segmenter is
+   * @return The {@link MediaType} this {@link Segmenter} is for.
+   */
+  MediaType getMediaType();
```

Comment:
I'd argue that a segmenter can work with more than only one media type and therefore, this should be a collection, ideally a set.

Reply:
That is a valid point. However, from my point of view it would be an interesting approach to derive the media type (by some registry?) from the actual type argument of the segmenter.
For now, I'm happy to make this change!

Reply:
I don't see how you would want to derive this, as it is dependent on the actual logic of the segmenter. The most trivial case would be the segmenter that just takes the entire object as one segment, that works for any type. Since we only have a small number of these segmenters anyway, it's easier for them to declare what types they support rather than trying to build something elaborate and possibly error prone so they don't have to.